apps/token/src/routes/governance/components/proposal/proposal.tsx

```diff
@@ -77,6 +77,15 @@ export const Proposal = ({ proposal }: ProposalProps) => {
         <div className="mb-8">
           <ProposalChangeTable proposal={proposal} />
         </div>
+        {proposal.terms.change.__typename === 'NewAsset' &&
```

Comment:
Chore (blocking) there's a `proposalType` in the file here, you can use that for this check

Reply:
Boo, ok fair dos